AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Notifications/InitialRequestConfirmationsSentHandlerShould.cs

```diff
@@ -19,15 +18,21 @@ public class InitialRequestConfirmationsSentHandlerShould : InMemoryContextTest
         [Fact]
         public async Task UpdateRequestStatusToPendingConfirmation()
         {
+            var dateTimeNow = DateTime.Now;

```

Comment:
To match the behavior of Itineraries in the system, this should be DateTime.Today